### PR TITLE
fix(freeze): Remove/replace freezing request

### PR DESCRIPTION
**Pull request by domsteinbach**
resolves DEV-1562

### CODE REVIEW

File: apps/dsp-app/src/app/workspace/resource/representation/representation.service.ts [hunk 1]
```diff
@@ -78,6 +57,20 @@ export class RepresentationService {
         );
     }
 
+    /**
+     * return the jwt token from the session to authenticate
+     * @return the token
+     */
+    public getTokenFromLocalStorage(): string {
```
Comment (mdelez) [inline apps/dsp-app/src/app/workspace/resource/representation/representation.service.ts:64]: This is the same method that you have below but is private. Should it be public or private?

File: apps/dsp-app/src/app/workspace/resource/representation/document/document.component.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
     <pdf-viewer
         class="pdf-viewer"
-        [src]= "{ url: src.fileValue.fileUrl, withCredentials: true }"
+        [src]= "{ url: src.fileValue.fileUrl,  withCredentials: true }"
```
Comment (mdelez) [inline apps/dsp-app/src/app/workspace/resource/representation/document/document.component.html:13]: does `withCredentials` have a fear of commas? :)

**Review by mdelez — APPROVED** [review]

File: apps/dsp-app/src/app/workspace/resource/representation/document/document.component.html (hunk 2 above)
Comment (domsteinbach) [reply apps/dsp-app/src/app/workspace/resource/representation/document/document.component.html:13]: I just needed some space ;-)

File: apps/dsp-app/src/app/workspace/resource/representation/representation.service.ts (hunk 1 above)
Comment (domsteinbach) [reply apps/dsp-app/src/app/workspace/resource/representation/representation.service.ts]: Damn, was due to merging. (I changed it to public in order the pdf viewer could possibly also send the token.)

**Pull request merged**